src/main/java/com/vadmack/mongodbtest/entity/Project.java

```diff
@@ -20,4 +20,17 @@
     private Long id;
 
     private String name;
+
+    private Long ownerId;
+
+    public void setOwnerId(Long ownerId) {
+        validateOwnerId();
+        this.ownerId = ownerId;
+    }
+
+    private void validateOwnerId() {
```

Comment:
The name of the method is confusing. Maybe `checkOwnerIdBeforeSet` would be better?